kibana-reports/public/components/report_definitions/report_trigger/report_trigger.tsx

```diff
@@ -108,28 +90,21 @@ export function ReportTrigger(props) {
   const [monitor, setMonitor] = useState(AVAILABLE_MONITOR_OPTIONS[0].value);
   const [trigger, setTrigger] = useState(AVAILABLE_TRIGGER_OPTIONS[0].value);
 
-  const handleReportTriggerType = (e: React.SetStateAction<string>) => {
+  const handleReportTriggerType = (e: string) => {
     setReportTriggerType(e);
-    trigger_schema['trigger_type'] = TRIGGER_OPTION_MAP[e];
   };
 
-  const handleScheduleRequestTime = (e: React.SetStateAction<string>) => {
-    setScheduleRequestTime(e);
-    trigger_params['schedule_type'] = SCHEDULE_OPTION_MAP[e];
-    if (e === 'nowOption') {
-      reportDefinitionRequest['report_type'] = 'Download';
-    }
+  const handleScheduleType = (e: React.SetStateAction<string>) => {
+    setScheduleType(e);
   };
 
-  const handleFutureDateTimeSelect = (
-    e: React.SetStateAction<moment.Moment>
-  ) => {
-    setFutureDateTimeSelect(e);
-  };
+  // const handleFutureDateTimeSelect = (
```

Comment:
I think it should be safe to remove all `FutureDate` components since we can't support it

Reply:
will do